tests/schema/test_schema_generation.py

```diff
@@ -67,9 +68,11 @@ class Query:
 def test_custom_execution_context():
     class CustomExecutionContext(GraphQLExecutionContext):
         def build_response(
-            self, data: AwaitableOrValue[Optional[Dict[str, Any]]]
+            self,
+            data: AwaitableOrValue[Optional[Dict[str, Any]]],
+            errors: List[GraphQLError],
         ) -> AwaitableOrValue[ExecutionResult]:
-            result = cast(ExecutionResult, super().build_response(data))
+            result = cast(ExecutionResult, super().build_response(data, errors))
```

Comment:
Not related to this change, but why is this `cast` here?

Reply:
fixed 😊